src/Avalonia.Input/IPointerDevice.cs

```diff
@@ -12,5 +12,7 @@ public interface IPointerDevice : IInputDevice
         void Capture(IInputElement control);
 
         Point GetPosition(IVisual relativeTo);
+
+        void LayoutUpdated(IInputRoot root);
```

Comment:
I don't think `LayoutUpdated` is a good name for this: it suggests a change in the layout of the scene, but this isn't actually related to layout. Something like `SceneInvalidated` would be a better name I think.

Also I'm not really sure about putting this in the `IPointerDevice` interface. That interface is user-facing but this represents an implementation detail. Not really sure what a better approach would be. @jkoritzinsky what do you think?